lib/cdo/create_header.rb

```diff
@@ -16,24 +16,34 @@ class CreateHeader
     minecraft_aquatic: {
       image: "logo_minecraft_aquatic_square.jpg"
     },
+    spritelab: {
```

Comment:
Hey Kelby - nice work!
I'm making an update soon to the user header (adding Python Lab), and noticed that the `minecraft_designer` logo does not seem to be available for users age < 13 and not in a teacher section.
Locally:
<img width="967" alt="dev-header-minecraft-error" src="https://github.com/user-attachments/assets/0d30d4ba-edaf-4c00-9a92-9b2b4758fae7" />
This is not currently an issue on `prod` since `_limit_project_types` does not seem to be set correctly (will share with platform team). But thought we may want to update the minecraft image. Thanks!